app/assets/stylesheets/content.scss

```diff
@@ -1,8 +1,21 @@
 /************** CONTENT CONTAINER *******************************/
 
 #content {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.